pyro/infer/mcmc/util.py

```diff
@@ -331,7 +331,7 @@ def _find_valid_initial_params(model, model_args, model_kwargs, transforms, pote
 
 def initialize_model(model, model_args=(), model_kwargs={}, transforms=None, max_plate_nesting=None,
                      jit_compile=False, jit_options=None, skip_jit_warnings=False, num_chains=1,
-                     init_strategy=init_to_uniform):
+                     init_strategy=init_to_uniform, initial_params=None):
```

Comment:
Another way is to make a hack: if num_chains=0, we return initial_params=None. So we don't have to introduce this new arg.